trunk/source/WitnessParser/src/de/uni_freiburg/informatik/ultimate/witnessparser/YamlWitnessParser.java

```diff
@@ -0,0 +1,142 @@
+package de.uni_freiburg.informatik.ultimate.witnessparser;
```

Comment:
Add license header and authors (perhaps in other files as well, I didn't check); don't forget Katie